src/app/examples/crud/components/crud.component.ts

```diff
@@ -0,0 +1,105 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, NgForm } from '@angular/forms';
+
+import { takeUntil, map } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+import { Store, select } from '@ngrx/store';
+
+import { ROUTE_ANIMATIONS_ELEMENTS } from '@app/core';
+
+import { State } from '../../examples.state';
+import { Book } from '../books.model';
+import {
+  ActionBooksPersist,
+  AddOne,
+  DeleteOne,
+  UpdateOne,
+  ActionBookSelect
+} from '../books.actions';
+import {
+  selectBooks,
+  selectCurrentBook,
+  selectAllBooks
+} from '../books.selectors';
+
+@Component({
+  selector: 'anms-crud',
+  templateUrl: './crud.component.html',
+  styleUrls: ['./crud.component.scss']
+})
+export class CrudComponent implements OnInit {
+  private unsubscribe$: Subject<void> = new Subject<void>();
+
+  routeAnimationsElements = ROUTE_ANIMATIONS_ELEMENTS;
+
+  emptyBook: Book = {
```

Comment:
I think it will be cleaner if we would create a factory function to create an empty book.